practice/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Refer https://www.typescriptlang.org/tsconfig to add more Type checking(ex: noImplicitAny, strictNullChecks, strict,...)